node/forest_libp2p/src/behaviour.rs

```diff
@@ -336,4 +372,41 @@ impl ForestBehaviour {
     pub fn peers(&self) -> &HashSet<PeerId> {
         &self.peers
     }
+
+    /// Adds peer to bitswap peer set
+    pub fn connect(&mut self, peer_id: PeerId) {
```

Comment:
how is this different from `add_peer`? Shouldn't that bitswap connect just happen there?

Reply:
You're right. Ill call this function there!